src/main/java/com/lpvs/service/LPVSQueueService.java

```diff
@@ -271,7 +271,24 @@ public void processWebHook(LPVSQueue webhookConfig) {
             pullRequest = lpvsPullRequestRepository.saveAndFlush(pullRequest);
             log.error("Can't authorize commentResults() " + e);
             e.printStackTrace();
-            delete(webhookConfig);
+            int currentAttempts = webhookConfig.getAttempts();
+            if (currentAttempts < maxAttempts) {
+                webhookConfig.setAttempts(currentAttempts++);
+                try {
+                    addFirst(webhookConfig);
+                } catch (InterruptedException e1) {
+                    log.warn("Failed to update Queue element");
+                    e1.printStackTrace();
+                }
+                queueRepository.save(webhookConfig);
+            } else {
+                log.warn(
+                        "Maximum amount of processing webhook reached for pull request: "
+                                + pullRequest.getId()
+                                + " "
+                                + pullRequest.toString());
```

Comment:
This line is strange. We have no custom toString() for pullRequest. 
It's better to print some meaningful info or implement custom toString().

Reply:
I've replaced to pull request URL for easy access not by ID.